src/modules/launcher/Plugins/Microsoft.Plugin.Shell/Main.cs

```diff
@@ -13,6 +13,7 @@
 using System.Reflection;
 using System.Windows.Input;
 using ManagedCommon;
+using Microsoft.Plugin.Folder.Sources;
```

Comment:
This shouldn't be here. One plugin should not depend on another

Reply:
Done.